lib/modules/manager/kotlin-script/index.ts

```diff
@@ -0,0 +1,9 @@
+import { MavenDatasource } from '../../datasource/maven';
+
+export { extractPackageFile } from './extract';
+
+export const defaultConfig = {
+  fileMatch: ['^.*\\.main\\.kts$'],
```

Comment:
```suggestion
  fileMatch: ['^.+\\.main\\.kts$'],
```

`.main.kts` should be disallowed 😉